benchmarks/resnet50/run-benchmarks.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -xe
```
To detect errors in CI?